marklogic-client-api/src/test/java/com/marklogic/client/impl/OpticPlanSearchOptionsTest.java

```diff
@@ -0,0 +1,53 @@
+package com.marklogic.client.impl;
+
+import com.marklogic.client.DatabaseClient;
+import com.marklogic.client.expression.PlanBuilder;
+import com.marklogic.client.row.RowManager;
+import com.marklogic.client.row.RowRecord;
+import com.marklogic.client.row.RowSet;
+import com.marklogic.client.test.Common;
+import com.marklogic.client.test.junit5.RequiresML12;
+import com.marklogic.client.type.PlanSearchOptions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import java.util.Iterator;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@ExtendWith(RequiresML12.class)
+class OpticPlanSearchOptionsTest {
```

Comment:
I'd like this to be similar to the existing `NewOpticMethodsInElevenDotOneTest`, which is in the `test.rows` package as well with all the other Optic tests. 

Are all of these only in 12 though - none of these new features are in 11.3? If so, still put this into `test.rows` and call it `FromSearchWithOptionsTest` as we'll probably have too many new things in 12 to put them into one generic test. 

Also, extend `AbstractOpticUpdateTest`, that provides a number of helpers to make the test easier to write.

Reply:
According to Aaron, these changes are only being made on the "develop" and "release-12.0.0-ea1" branches, which means they are only in MarkLogic 12+.